tests/React.Tests/Mvc/HtmlHelperExtensionsTests.cs

```diff
@@ -34,8 +35,12 @@ private Mock<IReactEnvironment> ConfigureMockEnvironment(IReactSiteConfiguration
 		public void ReactWithInitShouldReturnHtmlAndScript()
 		{
 			var component = new Mock<IReactComponent>();
-			component.Setup(x => x.RenderHtml(false, false, null)).Returns("HTML");
-			component.Setup(x => x.RenderJavaScript()).Returns("JS");
+
+			component.Setup(x => x.RenderHtml(It.IsAny<TextWriter>(), false, false, null))
```

Comment:
Thanks for updating the tests. Can you add back a test for RenderHtml that doesn’t use a TextWriter to show that the existing method still works?